shared/actions/tracker.js

```diff
@@ -97,13 +98,22 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
 
 export function getProfile (username: string): TrackerActionCreator {
   return (dispatch, getState) => {
+    dispatch({type: Constants.updateUsername, payload: {username}})
```

Comment:
It's used in the incoming call map for trackers. It inits the tracker store

On Friday, June 24, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/actions/tracker.js
> <https://github.com/keybase/client/pull/3187#discussion_r68473863>:
>
> > @@ -97,13 +98,22 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
> >
> >  export function getProfile (username: string): TrackerActionCreator {
> >    return (dispatch, getState) => {
> > +    dispatch({type: Constants.updateUsername, payload: {username}})
>
> I don't understand what this does? It's not used anywhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3187/files/18bdb78f6f2f2c26e2a7609d0fc46f83fd641170#r68473863>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhU60oR9sOa61Mel-nxVZ-rjv1fBlWks5qPGAJgaJpZM4I9TZw>
> .
>